tools/ci-cdk/canary-lambda/README.md

```diff
@@ -12,6 +12,29 @@ deployed SDK is functioning correctly. Similarly, it can be used to verify
 the previous version of the SDK continues to work after the deployment
 of the new version.
 
+Running the canary locally
+--------------------------
+For testing, it's helpful to be able to run the canary locally. To accomplish this, you first need to generate a
+Cargo.toml:
+
+```bash
+cd ../canary-runner
+# to use a version of the SDK, use `--sdk-version` instead
+cargo run -- build-bundle \
+  --sdk-path ../../../aws/sdk/build/aws-sdk/sdk/ \
+  --canary-path ../canary-lambda \
+  --manifest-only
+```
+
+Next, you can use `cargo run` in this directory in `--local` mode to invoke the canary:
```

Comment:
First time reading through this, it sounded like you want `cargo run -- --local` to take place in `canary-runner` rather than in `canary-lambda`. I guess "this directory" means the directory of this README file?